galsim/config/output_datacube.py

```diff
@@ -97,13 +98,16 @@ def getNImages(self, config, base, file_num):
             raise AttributeError("Attribute output.nimages is required for output.type = MultiFits")
         return galsim.config.ParseValue(config,'nimages',base,int)[0]
 
-    def writeFile(self, data, file_name):
+    def writeFile(self, data, file_name, config, base, logger):
```

Comment:
Yes.  For more consistency on the call syntax compared to other similar functions.  And so subclasses can use them.  I didn't actually need them for my FocalPlane output type, but when writing that class, I realized that some of the signatures were probably insufficiently flexible.